src/Page/TextLayer.tsx

```diff
@@ -184,7 +184,7 @@ export default function TextLayer() {
 
     layer.innerHTML = '';
 
-    const textContentSource = page.streamTextContent();
+    const textContentSource = page.streamTextContent({ includeMarkedContent: !customTextRenderer });
```

Comment:
Solution to the customTextRenderer problem. No change of behaviour if you use a custom render function.

Reply:
What's "customTextRenderer problem"?

Reply:
Well previously the custom text renderer had a 1-to-1 ratio of elements to text content. Now it's a bit different as the structure has more nested stuff within. I think the safest option is to just disable the accessibility layer if there is a custom renderer